mcs/class/Mono.Compiler/Makefile

```diff
@@ -6,7 +6,7 @@ LIBRARY_WARN_AS_ERROR = yes
 
 KEYFILE = $(LIBRARY_SNK)
 
-LIB_REFS =
+LIB_REFS = System
```

Comment:
we need to check if this is a problem when using the managed JIT as runtime compiler.